CODEOWNERS

```diff
@@ -0,0 +1,30 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+
+# Luke Sneeringer is the default owner for anything not explicitly taken by
+# someone else.
+*                             @lukesneeringer
+
+# Dave Gramlich is the primary author of the scripts that generate the
+# documentation and run test scripts.
+docs/*                        @callmehiphop
+scripts/*                     @callmehiphop
+
+# Dave Gramlich and Stephen Sawchuk are the primary authors and share
+# responsibility for most libraries and tests.
+packages/*                    @callmehiphop @stephenplusplus
+system-test/*                 @callmehiphop @stephenplusplus
+test/*                        @callmehiphop @stephenplusplus
+
+# Ali Sheikh is the primary maintainer of the Logging extension libraries.
```

Comment:
@ofrobots is there anyone else I should tag for these libraries / is there better wording to use?

Reply:
Can we do Github teams? If so @GoogleCloudPlatform/node-team.

Reply:
I think so-- cc @lukesneeringer on if that matches the intent of this file.

Reply:
Try it. If it does not work, we can change it. :-)

Reply:
Do we want to replace the @ofrobots mention with that team?

Also, I think that means the entirety of that team would be pinged for all PRs that relate to that code. Is that what we want? I don't know how many people are on that team, but I can imagine their getting pretty annoyed with the frequency of pinging on code they didn't write.

Reply:
The team is ultimately responsible for the code, so it is appropriate to ping the team IMO.

Reply:
It's cool with me if that's the way this is supposed to go. Out of curiosity, how many people are on that team?

Reply:
Also, should that team be in the catch-all as well?